l10n_es_aeat_partner_check/models/res_partner.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2019 Acysos S.L.
+# Copyright 2021 Landoo Sistemas de Informacion SL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import requests
+
+from odoo import _, api, fields, models
+
+RESULTS = [
+    ("NO IDENTIFICADO", _("No identificado")),
+    ("IDENTIFICADO", _("Identificado")),
+    ("NO PROCESADO", _("No procesado")),
+    ("NO IDENTIFICABLE", _("No identificable")),
+    ("IDENTIFICADO-BAJA", _("Identificado, baja")),
+    ("IDENTIFICADO-REVOCADO", _("Identificado, revocado")),
+]
+TYPES = [
+    ("sales_equalization", _("Régimen de recargo de equivalencia")),
+    ("standard", _("Régimen estándar")),
+]
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.depends("aeat_partner_name", "name")
+    def _compute_data_diff(self):
+        for partner in self.filtered("aeat_partner_check_result"):
+            if partner.aeat_partner_check_result != "NO IDENTIFICABLE":
+                # Odoo don't support two space between name. Some names
+                # in AEAT has two space instead one
+                partner.aeat_data_diff = False
+                if partner.aeat_partner_name:
+                    if partner.name != partner.aeat_partner_name.replace("  ", " "):
+                        partner.aeat_data_diff = True
+
+    aeat_partner_check_result = fields.Selection(
+        selection=RESULTS, string="Check Result", readonly=True
+    )
+    aeat_partner_vat = fields.Char(string="AEAT VAT", readonly=True)
+    aeat_partner_name = fields.Char(string="AEAT Name", readonly=True)
+    aeat_data_diff = fields.Boolean(
+        string="Data different", compute="_compute_data_diff", store=True
+    )
+    aeat_last_checked = fields.Datetime(string="Latest AEAT check", readonly=True)
+    aeat_partner_type = fields.Selection(
+        string="Partner type", selection=TYPES, readonly=True
+    )
```

Comment:
If you are refering to [v17 wiki migration doc](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0), I think this is not the case. This fields are filled by the AEAT response call and does'nt make sense to import this data from another source

Reply:
Yes, i'm referin to the v17 migration documentation that it says:
If a field is marked as `readonly=True` on the model, it won't be possible to import it through the Odoo import tool, so avoid it as possible and define the `readonly` attribute on the views instead.

Reply:
Yes, that's it. But doesn't make sense to make this fields importable or writable from any source that is not the response from AEAT, because it will miss his purpose of get the exact data registered on the AEAT database.